editor/src/components/editor/store/store-deep-equality-instances.ts

```diff
@@ -1635,7 +1632,7 @@ export function SpecialSizeMeasurementsKeepDeepEquality(): KeepDeepEqualityCall<
       parentFlexDirectionResult &&
       parentJustifyContentEquals &&
       parentFlexGapEquals &&
-      parentPaddingEquals &&
+      parentPaddingEquals.areEqual &&
```

Comment:
These are real serious bugs: because of truthiness we did not realize that these are object values, so they are always true! Thankfully, the `@typescript-eslint/strict-boolean-expressions` rule caught them.